scripts-dev/lint.sh

```diff
@@ -0,0 +1,3 @@
+isort -y -rc synapse tests scripts-dev scripts
+flake8 synapse tests
+python3 -m black .
```

Comment:
doesn't this end up blacking everything, including virtualenvs?

Reply:
Yes, that is not good. Do we really need to black more than `synapse/` and `tests/`?